blueprints/module-unification-app/files/src/resolver.js

```diff
@@ -0,0 +1,13 @@
+import Resolver from 'ember-resolver/resolvers/glimmer-wrapper';
```

Comment:
Changing this to `ember-resolver/resolvers/fallback` _should_ allow "classic" addons to continue to work within module unification apps (and therefore fix `{{welcome-page}}` without modification). Can you test to confirm?

Reply:
Thanks, I've made the change and confirmed that this works